src/wallet/rpcwallet.cpp

```diff
@@ -1455,7 +1463,7 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
             "listtransactions ( \"account\" count from includeWatchonly)\n"
             "\nReturns up to 'count' most recent transactions skipping the first 'from' transactions for account 'account'.\n"
             "\nArguments:\n"
-            "1. \"account\"    (string, optional) DEPRECATED. The account name. Should be \"*\".\n"
+            "1. \"address\"    (string, optional)  The address name.\n"
```

Comment:
Following offline discussions, we should probably add the possibility to specify an address, rather than substitute account specifications with addresses.
Please @MarcoOl94, restructure the code to keep accounts as is and add addresses